project/app/main.py

```diff
@@ -238,12 +233,15 @@ def listify(text):
     df = df[[
         'id', 'state', 'city',
         'date', 'title', 'description',
-        'links', 'geocoding'
+        'links', 'lat', 'long'
```

Comment:
'description' and 'links' should be 'desc' and url' unless these are changed in the JSONified file by backend or elsewhere